sdk/bin/prepare.sh

```diff
@@ -21,12 +21,12 @@ if [ -d "./proto" ] ; then
   echo "Protocol already built ('npm run clean' first to fetch and compile again)"
   echo
 else
-  if [ -n "$PROXY_SNAPSHOT_DIRECTORY" ]; then
-     # Use local proxy and sdk sources, useful for development, point PROXY_SNAPSHOTS_DIRECTORY to the local
-     # proxy project source directory
-     echo "Using snapshot of proxy and sdk protocols from '$PROXY_SNAPSHOT_DIRECTORY'"
-     cp -rf $PROXY_SNAPSHOT_DIRECTORY/protocols/proxy/src/main/protobuf/* ./proto/
-     cp -rf $PROXY_SNAPSHOT_DIRECTORY/protocols/sdk/src/main/protobuf/* ./proto/
+  if [ -n "$RUNTIME_SNAPSHOT_DIRECTORY" ]; then
+     # Use local runtime and sdk sources, useful for development, point RUNTIME_SNAPSHOT_DIRECTORY to the local
+     # runtime project source directory
+     echo "Using snapshot of proxy and sdk protocols from '$RUNTIME_SNAPSHOT_DIRECTORY'"
+     cp -rf $RUNTIME_SNAPSHOT_DIRECTORY/protocols/proxy/src/main/protobuf/* ./proto/
+     cp -rf $RUNTIME_SNAPSHOT_DIRECTORY/protocols/sdk/src/main/protobuf/* ./proto/
```

Comment:
Where do we set this ENV VAR `RUNTIME_SNAPSHOT_DIRECTORY` ? I could not find it.. jsut making sure we update that piece as well

Reply:
My understanding is that this done manually for local development when you want to develop against an unreleased Runtime version with sources on your machine.

Reply:
ah right, makes sense 👍